### DISCUSSION TITLE
(Game Issue) Steambot Chronicles loading and save screen not showing properly

### DISCUSSION THREAD

**Discussion by RaptorZX4** [PCSX2/pcsx2#11577] [General]
in Steambot Chronicles, when you reach a loading screen transition, the screen is supposed to gradually turn into a cel-shading style colors, sometimes looking Sepia. 
While in the emulator, it doesn't show this at all, it just very slightly blurs the game while it loads. And the save screen doesn't display properly as well. As far as i know, it's been like this for a really long time and never been fixed. I know this problem doesn't break the game, but i do hope it's gonna get fixed soon.

Save Screen in PCSX2:
![Clipboard_07-17-2024_01](https://github.com/user-attachments/assets/90407b1c-8782-407f-8e97-9ebf4638ba59)

Loading screen in PCSX2:
![Clipboard_07-17-2024_02](https://github.com/user-attachments/assets/4ea3e4b1-b23e-45f0-9c05-fa8ebee7e1d0)

This is what it's supposed to look like (screenshots taken from videos on Youtube.)
Save Screen:
![Clipboard_07-17-2024_04](https://github.com/user-attachments/assets/ed37eeb8-8f71-4d06-a4ad-da6f68809a2f)

Loading Screen:
![Clipboard_07-17-2024_03](https://github.com/user-attachments/assets/5f7ff80c-bcaf-466b-82d4-b7df18b2c7c7)

**Comment by F0bes** [comment]
We don't use GitHub discussions for support. Please visit our discord or forums.
https://discord.com/invite/TCz3t9k
https://forums.pcsx2.net/
